build/Foundation/dll/Foundation.def

```diff
@@ -874,4 +874,11 @@ LIBRARY Foundation
         ; MainDispatcher
         ScheduleMainRunLoop
         ScheduleMainRunLoopAsync
-        SetupMainRunLoopTimedMultipleWaiter
\ No newline at end of file
+        SetupMainRunLoopTimedMultipleWaiter
+
+        ;UTTypeHelpers (Internal)
+        _UTTypeCreatePreferredIdentifierForTag
+        _UTTypeCreateAllIdentifiersForTag
```

Comment:
Talked to dustin offline a bit. I think he is correct (per usual) and the "Core" set of frameworks are below foundation. It means I will need to adjust my packaging to include them in the"Core" package but it is almost certainly more correct and cleaner overall than adding these privates here. Sorry for giving you bad advice.

---
In reply to: [95276230](https://github.com/Microsoft/WinObjC/pull/1662#discussion_r95276230) [](ancestors = 95276230,95276171)